chapters/spec-licenses.md

```diff
@@ -1,6 +1,104 @@
-# Annex J Creative Commons Attribution License 3.0 Unported (Informative)
+# Annex J SPDX Specification Licenses (Informative)
 
-**License**
```

Comment:
Propose we add an introduction section to provide some context for each licenses - seen a lot of people confused which license applies to SPDX

- Community-Spec-1.0 - License applicable to specification
- CC-BY-3.0 - License applicable to specification for all version before 2.3
- CC-1.0 - License applicable to SPDX license list including license identifiers

Reply:
I think this is addressed as to Community-Spec-1.0 and CC-BY-3.0 in the edits I've proposed to both the LICENSE file in the repo, as well as the lead-in in the chapters/index.md file for what will be populated in the spec itself. CC0-1.0 is not the license for the SPDX license list or identifiers.